salt/tests/unit/modules/files/test_metalk8s_checks.yaml

```diff
@@ -54,7 +56,13 @@ node:
       interface and route for this range (for details, see
       https://github.com/kubernetes/kubernetes/issues/57534#issuecomment-527653412).
 
-  # 7. Error: Combined errors
+  # 7. Error: fs_check
+  - <<: *node_nominal
+    fs_check_ret: "Containerd XFS filesystem (/var/lib/containerd) has ftype=2 expected 1"
+    expect_raise: True
+    result: 'Node my_node_1: Containerd XFS filesystem \(/var/lib/containerd\) has ftype=2 expected 1'
+
+  # 8. Error: Combined errors
```

Comment:
You should add an fs_check error in this one as well so that we ensure that we properly get all messages at once when you have several issues